distribution/lib/Standard/Database/0.0.0-dev/src/Data/DB_Column.enso

```diff
@@ -81,17 +81,17 @@ type DB_Column
 
     ## ICON data_input
        Converts this column into a single-column table.
-    to_table : Table
+    to_table : DB_Table
     to_table self =
-        Table.Value self.name self.connection [self.as_internal] self.context
+        DB_Table.Value self.name self.connection [self.as_internal] self.context
 
     ## ALIAS metadata
        GROUP Standard.Base.Metadata
        Returns a Table describing this column's contents.
 
        The table behaves like `Table.info` - it lists the column name, the count
        of non-null items and the value type.
-    info : Table
+    info : DB_Table
     info self = self.to_table.info
```

Comment:
This is actually returning an in-memory table, not `DB_Table.

Suggestion: you could add opt-in return type checks - then if there's misalignment in the types, it will fail in tests.